spec/requests/cache_spec.rb

```diff
@@ -118,33 +118,33 @@ module DisabledAnonymousAPI
 end
 
 describe 'Caching behavior' do
-  shared_examples 'cachable response' do
-    it 'does not set cookies' do
+  shared_examples 'cachable response' do |http_success: false|
```

Comment:
I could also see this being `http_status: false` and passing in a numeric code or symbol instead of just true/false on success ... but as of now all the places that use this are only using it for 200/success.